documentation/modules/security/proc-accessing-kafka-using-ingress.adoc

```diff
@@ -3,43 +3,60 @@
 // assembly-configuring-kafka-listeners.adoc
 
 [id='proc-accessing-kafka-using-ingress-{context}']
-= Accessing Kafka using ingress
+= Accessing Kafka using ingresses
 
-This procedure shows how to access a Strimzi Kafka cluster from an external client outside of Kubernetes using Nginx Ingress.
+[role="_abstract"]
+Use an {NginxIngressController} to access a Strimzi Kafka cluster from clients outside the Kubernetes cluster. 
+The Nginx Ingress Controller supports connections from HTTP and HTTPS-based applications.
 
-To connect to a broker, you need a hostname (advertised address) for the Ingress _bootstrap address_,
-as well as the certificate used for authentication.
+To be able to use an Nginx Ingress Controller, add configuration for an `ingress` type listener in the `Kafka` custom resource. 
```

Comment:
There are two Nginx Ingress controllers. The one we support is this one from Kubernetes: https://github.com/kubernetes/ingress-nginx ... AFAIK, it is not always the same as the Nginx Ingress Controller from NGINX company (https://docs.nginx.com/nginx-ingress-controller/). So we need to be clear about this in all places.

```suggestion
To be able to use a Kubernetes Nginx Ingress Controller, add configuration for an `ingress` type listener in the `Kafka` custom resource. 
```